test/test/tearoff.cpp

```diff
@@ -215,6 +229,11 @@ TEST_CASE("tearoff")
         REQUIRE(S_OK == persist->GetClassID(&result));
         REQUIRE(winrt::is_guid_of<IPersist>(result));
 
+        winrt::com_array<winrt::guid> iids = winrt::get_interfaces(closable);
```

Comment:
This test file adds tearoff IIDs to both `RuntimeType` and `InteropType` but only has test cases for `InteropType`.  It would be nice to also check the IIDs of `RuntimeType`.